data/campaigns/Eastern_Invasion/scenarios/13_Spoils_of_War.cfg

```diff
@@ -22,7 +22,12 @@
 
     [story]
         [part]
-            title= _ "<b><span color='#ff0000'>Part III: Wesnoth at War</span></b>"
+            #po: newlines at the beginning help to (depending on the screne size) center this vertically. The space at the end stops the last letter from getting cut off
```

Comment:
The latter part sounds like this should be a bug report. Surely it is not expected that text gets cut off.